search/indexing_api.py

```diff
@@ -109,12 +110,16 @@ def index_program_enrolled_users(program_enrollments, chunk_size=100):
     Returns:
         int: Number of indexed items
     """
-    return _index_chunks(
+    num_indexed = _index_chunks(
         (serialize_program_enrolled_user(program_enrollment) for program_enrollment in program_enrollments),
         USER_DOC_TYPE,
         chunk_size=chunk_size,
     )
 
+    for program_enrollment in program_enrollments:
+        send_automatic_emails(program_enrollment)
```

Comment:
this seems like a _really_ good candidate for an asynchronous task. we already have an issue open about our poor indexing performance. this adds quite a lot of work to the indexing of every single user - a `percolate` call to ES, a query to `PercolateQuery`, a query to `AutomaticEmail`, potentially a Mailgun call, etc.